.github/workflows/on_run_create_jira_flaky_test.yml

```diff
@@ -0,0 +1,66 @@
+name: On Run Create Jira for Flaky Test
+on:
+   workflow_run:
+     workflows: [On PR Open or Push Do Build and Test]
+     types: [completed]
+jobs:
+   download:
+      name: Downloading Flaky test report
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v4
+
+        - name: Install xmlstarlet tool
+          shell: bash
+          run: |
+             sudo apt-get update
```

Comment:
Do we need to update everything?

It would be good to add a comment explaining that xmlstarlet is required by `track_flaky_tests.sh`

Reply:
seems it avoids troubles: [Installing software on Ubuntu runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/customizing-github-hosted-runners#installing-software-on-ubuntu-runners)

ok I'll add the comment